### PR TITLE
qa/tasks/ceph2: add support for shell, packaged ceph-daemon

### PR DESCRIPTION
**Pull request by liewegas**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by liewegas** [comment]
passing my local tests (at least until it hits https://github.com/ceph/ceph/pull/31952)

File: qa/suites/rados/ssh/start.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 tasks:
 - install:
+    extra_packages: [ceph-daemon]
```
Comment (jdurgin) [inline qa/suites/rados/ssh/start.yaml]: should this be an override in packaged.yaml?

Comment (liewegas) [reply qa/suites/rados/ssh/start.yaml]: pushed another patch to do this

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Comment by gqqnbig** [comment]
I learned the packaged mode from https://lists.ceph.io/hyperkitty/list/ceph-users@ceph.io/thread/3ED4NZV436MOSTXC2TSO6BPZL3NSXBDD/ and found this PR. However, I failed to find any cues on the [documentation](https://docs.ceph.com/en/latest/search/?q=packaged&check_keywords=yes&area=default).

What is the status of the feature? Is it stable and sort of first class support?

**Pull request merged**